src/app/ubs/ubs/components/ubs-order-details/ubs-order-location-popup/ubs-order-location-popup.component.scss

```diff
@@ -36,6 +36,52 @@
     }
   }
 
+  .form-control {
+    background-color: var(--white);
+  }
+
+  .mat-input-element {
+    padding: 7px;
+    height: 50px;
+  }
+
+  ::ng-deep .mat-form-field-infix {
```

Comment:
Add **:host** before very **::ng-deep** to avoid rewrite style globally
Example: `:host ::ng-deep .mat-form-field-infix {}`

Reply:
Changed code, removed ::ng-deep.